From: Sean Whitton <spwhitton@spwhitton.name>
Date: Fri, 26 Feb 2016 09:19:11 -0700
Subject: fix autoload tests
Forwarded: not-needed

dash-el is currently outside of the dh_elpa infrastructure and so has no
autoloads file.  So hack the tests to use another.
---
 test/flycheck-test.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/test/flycheck-test.el
+++ b/test/flycheck-test.el
@@ -497,7 +497,7 @@
 
 (ert-deftest flycheck-autoloads-file-p/autoloads-with-backing-file ()
   :tags '(utility)
-  (flycheck-ert-with-file-buffer (locate-library "dash-autoloads")
+  (flycheck-ert-with-file-buffer (locate-library "shut-up-autoloads")
     (should (flycheck-autoloads-file-p))))
 
 (ert-deftest flycheck-autoloads-file-p/a-plain-file ()
@@ -3324,7 +3324,7 @@ See https://github.com/flycheck/flycheck
   ;; collect the autoloads of newly installed packages before writing the
   ;; autoloads file.  See `https://github.com/flycheck/flycheck/issues/45' and
   ;; `https://github.com/bbatsov/prelude/issues/253' for details.
-  (flycheck-ert-with-file-buffer (locate-library "dash-autoloads")
+  (flycheck-ert-with-file-buffer (locate-library "shut-up-autoloads")
     (should-not (flycheck-may-use-checker 'emacs-lisp))
     (should-not (flycheck-may-use-checker 'emacs-lisp-checkdoc))))
 
